### DISCUSSION TITLE
Can’t see the name files in his totality in “releases” because the name is large.

### DISCUSSION THREAD

**Discussion by Puzzolo95** [community/community#114310] [Mobile]
Tried in landscape but it freezes, tried to enter the app in landscape and when I reach the "releases with the file names it freezes
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.154.0 (125806213)
Device: iPhone 14 Plus (iOS 17.4.1)
</details>

**Comment by elisealix22** [comment]
Hi @Puzzolo95, thank you for writing in! I opened up an internal issue to look into this. Some questions that might help are team are: Do you know if you are on the release detail screen when it freezes? Or is it the list view? Also, if you have any screenshots you can share or any public examples that always helps. Either way, we will look into it!

**Reply by Puzzolo95** [reply]
Hello, I am in the releases section, under "assets"![image](https://github.com/community/community/assets/149212962/b022519e-687e-49dd-9380-d340b3797b9b)

**Comment by MrNiSso** [comment]
🔴Can u ratings my app and give me to /10🌟

https://play.google.com/store/apps/details?id=com.fd.food_recipe0
